superset-frontend/src/addSlice/AddSliceContainer.tsx

```diff
@@ -297,29 +313,38 @@ export default class AddSliceContainer extends React.PureComponent<
                   value={this.state.datasource}
                 />
                 <span>
-                  {t(
-                    'Instructions to add a dataset are available in the Superset tutorial.',
-                  )}{' '}
+                  <a
```

Comment:
Shouldn't this option only appear if the user has permission to add a dataset?

Reply:
I see the argument but I'd vote to also save this for when `addSlice` is brought into the SPA, because right now the `addSlice` React app doesn't pull in any permissions info.

Reply:
You just need to do the following change in `superset/views/chart` to include the permissions:

<img width="584" alt="Screen Shot 2022-06-17 at 4 34 16 PM" src="https://user-images.githubusercontent.com/70410625/174390577-09123370-56e2-482b-ba15-b7908be14147.png">

And they will be available in Redux:

<img width="341" alt="Screen Shot 2022-06-17 at 4 36 42 PM" src="https://user-images.githubusercontent.com/70410625/174390791-773ae4fa-cc1e-45c3-980b-65b77102e991.png">

Reply:
Oh great thanks!

Reply:
Done